.azure/pipelines/ci.yml

```diff
@@ -38,10 +63,16 @@ jobs:
     steps:
     - powershell: ./eng/scripts/CodeCheck.ps1 -ci
```

Comment:
If people run CodeCheck.ps1 locally, they would need to build NodeJS first. Otherwise during RefAssembly generation, the Components.Server will be missing the JS files which are embedded resources. Because I wanted to allow local runs of CodeCheck.ps1, I am opting to keep the behaviour where it builds NodeJS. As such, there's no need to add a dependency on the NodeJS job on the CI and copy its artifacts.